### PR TITLE
Fix language picker height

### PR DESCRIPTION
**Pull request by JohnONolan**
The language picker in main navigation opens at a very small height, and it isn't obvious that it can be scrolled, so it feels broken. Suggested change allows the dropdown to expand to fix content, making all languages easily discoverable. This change is for desktop viewports only, the mobile behaviour is unchanged (which already shows all languages). 

## Before:

Screenshot 2023-01-23 at 14 06 47@2x

## After:

Screenshot 2023-01-23 at 14 06 20@2x

### CODE REVIEW

**Review by AugustinMauroy — APPROVED** [review]

**Review by ghost — COMMENTED** [review]
LGTM：What about making it shorter? For the "After" img, it seems the menu of languages is a bit too long....Maybe we need half of the length with a scrolling bar...?

**Comment by ovflowd** [comment]
This is fixed on #4991. We cannot merge this now due to code freeze, and once #4991 gets merged I'm pretty sure this PR becomes obsolete...

**Comment by JohnONolan** [comment]
Ah, nice! I missed that — thanks for info

**Pull request closed without merging**
